aten/src/ATen/native/native_functions.yaml

```diff
@@ -7807,6 +7807,26 @@
   use_c10_dispatcher: full
   variants: function
 
+- func: fft_fftfreq(int n, float d=1.0, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
would "window_length" be more consistent with the existing fft functions, as opposed to "n" as the first argument?

Reply:
`n` is the name of the second argument to `fft`. So, I think that in this case being consistent with numpy is more important.